mackerel-plugin-mongodb/test.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
```

Comment:
This file has shebang but its permission is `0644`. Should we set this to `0755` ?